ui/apps/platform/cypress/integration/accessControl/accessControlPermissionSets.test.js

```diff
@@ -219,18 +219,20 @@ describe('Access Control Permission sets', () => {
         );
         cy.get(getAccessLevelSelectForResource('Detection')).should('contain', 'Read Access');
 
-        // Zero-based index for Image instead of ImageComponent, ImageIntegration, WatchedImage.
-        cy.get(getReadAccessIconForResource('Image', 0)).should(
+        // Zero-based index for Image because:
+        // preceded by BaseImage, BaseImageLayer
+        // followed by ImageComponent, ImageIntegration, WatchedImage.
+        cy.get(getReadAccessIconForResource('Image', 2)).should(
```

Comment:
Ah, I see the issue and how this fixes it. The helper function still seems fragile. It could very well break again due to a backend change. Is it possible to do an exact match? If we can, I'd recommend that. If it's a bigger change or requires more investigation, I'm ok with approving and coming back to it.

Reply:
Yeah, we have examples of `contains` method with RegExp for complete match.

Looking at the complicated selector, I need to rethink it, and maybe even the test itself.

Wow, in such a hurry that I did not even notice `ImageComponent` and `ImageIntegration` are long gone!